c-api/compat-5.3.c

```diff
@@ -393,7 +413,7 @@ COMPAT53_API int luaL_fileresult (lua_State *L, int stat, const char *fname) {
 }
 
 
-static const char* compat53_f_parser_handler (lua_State *L, void *data, size_t *size) {
+static const char* compat53_f_parser_handler (lua_State *, void *data, size_t *size) {
```

Comment:
Why this change?

Reply:
clang and g++ on -Wall -Wpedantic and all of those other settings warns about unused parameters.